### DISCUSSION TITLE
Copilot is blocked in Ukraine now

### DISCUSSION THREAD

**Discussion by aleevsergey** [community/community#47641] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Did you forgot to say in the last update that Copilot blocked for Ukraine users even with paid subscription
This how you "support" our in difficult times?

https://github.com/orgs/community/discussions/47577

How to get refund?

**Comment by tuves** [comment, marked as answer]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.

**Comment by tuves** [comment]
We became aware that Copilot was unavailable in several countries following our efforts to restore access to Ukraine, and on further investigation, we identified the cause as originating from a change we deployed to block Copilot in trade-restricted countries. To prevent this from happening again, we've updated our reporting dashboards to better account for error rates due to trade restrictions and are implementing more robust testing of the trade restriction logic.
